### DISCUSSION TITLE
Can't get in maskrom mode

### DISCUSSION THREAD

**Discussion by Belpol** [ntherning/StorytelReaderMods#1] [Q&A]
First of all thank you for this amazing project. 
I tried the steps to get into the maskrom mode on a 2nd gen storytel-reader...but don't succeed. 
I connected a USB-A to USB-C. 
Pressed the orange button
Then connected the device side of the reader with USB-C
It appears for a few seconds in the hardware list, but then it disconnects from my pc (can hear the disconnect sound) and it boots into recovery mode...

Am i doing something wrong? 
Did Storytel patch it so it isn't possible anymore?

Thanks!

**Comment by ntherning** [comment]
Hi, thank you for giving it a try!

Which OS are you using?

I am keeping track of new updates from Storytel and AFAICT there haven't been any updates since late summer, before I started this project. My reader is on the latest version.

Are you certain the cable you are using is a data cable and not just a charging cable? For me I can get into maskrom mode using the white cable with orange connector which came with the 2nd gen Storytel Reader. From your description it sounds like it might be a charging cable? I found that when you connect the Reader to a charger and press the orange button while powering it on it enters recovery mode: https://github.com/ntherning/StorytelReaderMods#2nd-generation.

**Reply by EnCodecwtf** [reply]
Same problem new device.
I have tried different cables.
Different times holding the orange button difrent times putting in the cable.

Nothing happenes. Some times it appears for 20-30 sec in devicemanager but always disconnected after a few seconds and thrown in to recovery mode.
